w03-intro-backend-with-express/d4-dusk-mongoose/README.md

```diff
@@ -0,0 +1,218 @@
+
+	```javascript
+		var AddressBookSchema = new Schema({
+		    firstName: String,
+		    lastName: String,
+		    address: String
+		    phoneNumber: Number,
+		    email: String
+		    professionalContact: Boolean
+		});
+	```
+*With the above Schema, we can expect all of our Address Book entries would have a first name, last name, address, and email address in the form of Strings.  We can count on the phoneNumber to always be accepted, stored, and returned as a number.  Lastly, the boolean value of Professional Contact will always be a `true` or `false`*
+
+- **Model**: A model is a Schema that has been 'activated' with real data and is performing actions such as reading, saving, updating, etc.
+
+##Schema vs. Model
+
+>"In mongoose, a schema represents the structure of a particular document, either completely or just a portion of the document. It's a way to express expected properties and values as well as constraints and indexes. A model defines a programming interface for interacting with the database (read, insert, update, etc). So a schema answers "what will the data in this collection look like?" and a model provides functionality like "Are there any records matching this query?" or "Add a new document to the collection". ""
+
+-[Peter Lyons Apr 8 '14 at 23:53](http://stackoverflow.com/a/22950402)
+
+
+
+##RDBMS vs noSQL
+A great analogy from a fellow StackOverflow:
+> "NoSQL databases store information like you would recipes in a book. When you want to know how to make a cake, you go to that recipe, and all of the information about how to make that cake (ingredients, preparation, mixing, baking, finishing, etc.) are all on that one page.
+> 
+> SQL is like shopping for the ingredients for the recipe. In order to get all of your ingredients into your cart, you have to go to many different aisles to get each ingredient. When you are done shopping, your grocery cart will be full of all the ingredients you had to run around and collect.
+> 
+> Wouldn’t it be nicer if there was a store was organized by recipe, so you could go to one place in the store and grab everything you need from that one spot? Granted you’ll find ingredients like eggs in 50 different places, so there’s a bit of overhead when stocking the shelves, but from a consumer standpoint it was much easier/faster to find what they were looking for."  
+
+-[mgoffin, Jan 20 '13 at 19:15](http://stackoverflow.com/questions/14428069/sql-and-nosql-analogy-for-the-non-technical/14428221#14428221)  
+
+
+##Getting started with our own database!
+####Caution: At this point, I urge you to simply read and absorb the information provided to see the basic patterns of creating and implementing databases.  You do not need to create a new node project to successfully learn from tonight's readings.  We'll go over this at a comfortable pace tomorrow :)
```

Comment:
This line doesn't seem to fit since they didn't have this for reading last night?